firestore/integration_test.go

```diff
@@ -215,6 +227,34 @@ func deleteCollection(ctx context.Context, coll *CollectionRef) error {
 	return nil
 }
 
+func deleteDocument(ctx context.Context, docRef *DocumentRef, bulkwriter *BulkWriter) error {
```

Comment:
How does this relate to multi-DB support?

Reply:
Removed